pyface/ui/wx/python_shell.py

```diff
@@ -309,7 +309,7 @@ def Destroy(self):
         self.redirectStdin(False)
         builtins.raw_input = self.raw_input
         self.destroy()
-        super(PyShellBase, self).Destroy()
```

Comment:
This is a potentially buggy use of `super`

Reply:
~This is strange as I believe `PyShellBase` only has a `destroy` method not a `Destroy` method:
https://wxpython.org/Phoenix/docs/html/wx.py.shell.Shell.html#wx.py.shell.Shell.destroy~
Actually `Destroy` is defined much further up the inheritance tree: https://wxpython.org/Phoenix/docs/html/wx.Window.html#wx.Window.Destroy